opentelemetry-sdk/src/metrics/manual_reader.rs

```diff
@@ -76,10 +76,11 @@ impl MetricReader for ManualReader {
             // Only register once. If producer is already set, do nothing.
             if inner.sdk_producer.is_none() {
                 inner.sdk_producer = Some(pipeline);
-            } else {
-                global::handle_error(MetricsError::Config(
-                    "duplicate reader registration, did not register manual reader".into(),
-                ))
+            }  else {
+                otel_debug!(
+                    name: "ManualReader.RegisterPipeline.DuplicateRegistration",
```

Comment:
I think it's better to leave out the implementation details from the event names. We might or might not change this method name or we might rename pipeline to something else in the future.

```suggestion
                    name: "ManualReader.DuplicateRegistration",
```

Reply:
We should come up with some convention all these events would follow. For example, something like this:
`{crate}.{component}.{optional subcomponent}.{eventName}`

`Sdk.MetricReader.ManualReader.DuplicateRegistration`

Reply:
We are already adding crate name in the macro. Trying to follow the naming as 
`ModuleName.SecondLevel.OptionalThirdLevel.EventName`

second lvel could be struct / method in this module.

Reply:
Internal modules could still be moved around or refactored so I think we should rely on:
`component name (spec concepts such as MeterProvider, MetricReader, etc.) -> sub component(s) -> event name`